src/main/java/Job.java

```diff
@@ -0,0 +1,102 @@
+import org.json.JSONObject;
+import java.util.Date;
+
+public class Job {
+    private JSONObject measurementDesc;
+    private Date startTime;
+    private Date endTime;
+    private Date nextReset;
+    private int jobInterval; //in hrs
+    private int requiredNodeCount;
+    private int currentNodeCount;
+
```

Comment:
I think we need a jobId outside here.

Reply:
yeah I think thats also fine instead of looking into the Measurement Desc everytime

Reply:
@MelulekiDube pass a boolean and jobId so that i can mark a job as ```successful```